numba/cuda/mathimpl.py

```diff
@@ -60,9 +60,11 @@
 binarys += [('fmod', 'fmodf', math.fmod)]
 binarys += [('hypot', 'hypotf', math.hypot)]
 binarys += [('remainder', 'remainderf', math.remainder)]
+binarys += [('nextafter', 'nextafterf', math.nextafter)]
 
 binarys_fastmath = {}
 binarys_fastmath['powf'] = 'fast_powf'
+binarys_fastmath['nextafterf'] = 'fast_nextafterf'
```

Comment:
Looks like there is a fast version of `nextafter` for 64-bit operands too: https://docs.nvidia.com/cuda/libdevice-users-guide/__nv_nextafter.html#__nv_nextafter - I'm not 100% sure this will work (been a while since I thought about these implementations) but maybe:

```suggestion
binarys_fastmath['nextafterf'] = 'fast_nextafterf'
binarys_fastmath['nextafter'] = 'fast_nextafter'
```

Reply:
Additionally - if there are fastmath implementations of these functions, they should also be tested in `numba/cuda/tests/cudapy/test_fastmath.py` - you should be able to create a test by following the pattern used in the tests for other functions. If you're having trouble working out what patterns to check for, let me know and I'll see if I can help work out something appropriate.